content/applications/inventory_and_mrp/inventory/warehouses_storage/replenishment/resupply_warehouses.rst

```diff
@@ -2,69 +2,166 @@
-reordering rule. The location between the delivery order and the receipt is a transit location.
+Tick the :guilabel:`X: Supply Product from Y` checkbox, which is intended to be used with the |MTO|
+route or a reordering rule to replenish stock by moving the product from one warehouse to another.
+Proceed to the dedicated sections below to continue the process.
 
-.. image:: resupply_warehouses/resupply-receipts-from-reordering-rule.png
-   :align: center
-   :alt: A reordering rule automatically creates two receipts for stock between warehouses.
+MTO
+~~~
 
-.. image:: resupply_warehouses/second-warehouse-delivery-order.png
-   :align: center
-   :alt: A warehouse order for resupplying one warehouse's stock with another.
+To replenish products using the make to order method, go to the product form and ensure the
+:ref:`MTO route is unarchived <inventory/warehouses_storage/unarchive-mto>`, so it appears in the
+:guilabel:`Routes` section of the :guilabel:`Inventory` tab.
+
+With the resupply and |MTO| routes ticked, jump to the next :ref:`Resupply from another warehouse
+workflow <inventory/warehouses_storage/resupply-workflow>` section.
+
+.. example::
+   The product, sold at the warehouse, `Store`, is resupplied from the central warehouse, named
+   `YourCompany`. To replenish the product using |MTO|, the following routes are selected:
+
+   - :guilabel:`Store: Supply Product from YourCompany`
+   - :guilabel:`Replenish on Order (MTO)`
+
+   .. image:: resupply_warehouses/resupply-route.png
+      :align: center
+      :alt: Route setting which enables a product to resupplied from a second warehouse.
+
+Reordering rule
+~~~~~~~~~~~~~~~
+
+To replenish products using reordering rules, ensure the :guilabel:`X: Supply Product from Y` route
+is selected in the :guilabel:`Inventory` tab of the product form.
+
+In addition, create a reordering rule to automate replenishment by clicking the
+:guilabel:`Reordering Rules` smart button.
+
+Next, click :guilabel:`New`, and set:
+
+- :guilabel:`Location`: the stock location of the retail store. For example, `SHOP/Stock`.
+- :guilabel:`Route`: :guilabel:`X: Supply Product from Y`.
+- Configure the :guilabel:`Min Quantity` and :guilabel:`Max Quantity` to trigger automatic stock
+  transfers when inventory falls below the set threshold.
+
+.. seealso::
+   :doc:`reordering_rules`
+
+.. example::
+   A :ref:`0/0 reordering rule <inventory/warehouses_storage/zero-zero>` to replenish the shop's
+   warehouse is created, with the :guilabel:`Location` set to `SHOP/Stock`, and the
+   :guilabel:`Route` set to :guilabel:`Store: Resupply from YourCompany`.
+
+   .. image:: resupply_warehouses/reordering-rule.png
+      :align: center
+      :alt: Show reordering rule configurations.
+
+.. _inventory/warehouses_storage/resupply-workflow:
+
+Replenish one warehouse from another
+====================================
+
+After completing the setup, trigger the replenishment method using one of the many methods, such as:
```

Comment:
You say, "one of the many methods" but then list two. Are you unsure how many methods there are? If there are others, could they be enumerated (e.g. one of three methods) and then listed or linked to other documentation?